diff -aurN seahorse-0.9.3.orig/daemon/Makefile.am seahorse-0.9.3/daemon/Makefile.am
--- seahorse-0.9.3.orig/daemon/Makefile.am	2006-08-24 22:13:06.000000000 +0200
+++ seahorse-0.9.3/daemon/Makefile.am	2006-08-25 20:11:18.000000000 +0200
@@ -22,9 +22,11 @@
 	seahorse-service-crypto-bindings.h 
 
 if WITH_AGENT
-AGENT_LIB = $(top_builddir)/agent/libseahorseagent.a $(AGENT_LIBS)
+AGENT_LIB = $(AGENT_LIBS)
+AGENT_STATIC_LIB = $(top_builddir)/agent/libseahorseagent.a 
 else
 AGENT_LIB = 
+AGENT_STATIC_LIB = 
 endif
 
 if WITH_SHARING
@@ -40,9 +42,10 @@
     $(SHARING_SRCS)
 
 seahorse_daemon_LDADD = \
+    $(AGENT_STATIC_LIB) \
+    $(top_builddir)/libseahorse/libseahorse.a \
     $(AGENT_LIB) \
     $(top_builddir)/libcryptui/libcryptui.la \
-    $(top_builddir)/libseahorse/libseahorse.a \
     $(SEAHORSE_LIBS)
 
 man_MANS = seahorse-daemon.1
diff -aurN seahorse-0.9.3.orig/plugins/nautilus/Makefile.am seahorse-0.9.3/plugins/nautilus/Makefile.am
--- seahorse-0.9.3.orig/plugins/nautilus/Makefile.am	2006-08-24 22:13:08.000000000 +0200
+++ seahorse-0.9.3/plugins/nautilus/Makefile.am	2006-08-25 19:42:52.000000000 +0200
@@ -14,8 +14,8 @@
 seahorse_preferences_SOURCES = seahorse-pgp-preferences.c 
 
 seahorse_preferences_LDADD = \
-    $(top_builddir)/libcryptui/libcryptui.la \
     $(top_builddir)/libseahorse/libseahorse.a \
+    $(top_builddir)/libcryptui/libcryptui.la \
     $(SEAHORSE_LIBS)
     
 seahorse_tool_SOURCES = \
@@ -25,9 +25,9 @@
     seahorse-tool-progress.c
 
 seahorse_tool_LDADD = \
+    $(top_builddir)/libseahorse/libseahorse.a \
     $(SEAHORSE_LIBS) $(LIBCRYPTUI_LIBS) \
-    $(top_builddir)/libcryptui/libcryptui.la \
-    $(top_builddir)/libseahorse/libseahorse.a
+    $(top_builddir)/libcryptui/libcryptui.la
 
 # The new mime system
 mimedir = $(datadir)/mime/packages
