diff -ur seahorse-3.4.1.orig//pgp/seahorse-gpgme-keyring.c seahorse-3.4.1/pgp/seahorse-gpgme-keyring.c
--- seahorse-3.4.1.orig//pgp/seahorse-gpgme-keyring.c	2012-01-27 13:50:26.000000000 +0000
+++ seahorse-3.4.1/pgp/seahorse-gpgme-keyring.c	2012-07-23 17:55:58.026184013 +0000
@@ -859,7 +859,7 @@
 {
 	SeahorseGpgmeKeyring *self = SEAHORSE_GPGME_KEYRING (object);
 
-	g_clear_object (self->pv->actions);
+	g_clear_object (&self->pv->actions);
 	g_hash_table_destroy (self->pv->keys);
 
 	/* All monitoring and scheduling should be done */
