diff -urN seahorse-2.22.3/configure.in seahorse-2.22.3.new/configure.in
--- seahorse-2.22.3/configure.in	2008-05-26 20:13:10.000000000 +0200
+++ seahorse-2.22.3.new/configure.in	2008-07-10 01:18:02.682912385 +0200
@@ -506,10 +506,12 @@
             AC_SUBST([MOZILLA_HOME])
             AC_SUBST([MOZILLA_PREFIX])
             AC_SUBST([MOZILLA_INCLUDE_ROOT])
+            AC_SUBST([MOZILLA_XPCOM])
 
             case "$MOZILLA" in
             xulrunner) min_version=1.8 ;;
             *firefox) min_version=1.5 ;;
+            libxul-unstable) min_version=1.9 ;;
             *) AC_MSG_ERROR([Unsupported gecko "$gecko_cv_gecko"]) ;;
             esac
              
@@ -591,7 +593,7 @@
         
         if test "$with_epiphany_plugin" = "yes"; then
             PKG_CHECK_MODULES([MOZILLA_COMPONENT],
-            		          [${gecko_cv_gecko}-xpcom >= $min_version $gecko_cv_extra_pkg_dependencies],
+            		          [$MOZILLA_XPCOM >= $min_version $gecko_cv_extra_pkg_dependencies],
             		          [with_epiphany_plugin=yes],[with_epiphany_plugin=no echo "disabling epiphany plugin"])
         fi
         		          
diff -urN seahorse-2.22.3/m4/gecko.m4 seahorse-2.22.3.new/m4/gecko.m4
--- seahorse-2.22.3/m4/gecko.m4	2007-11-24 22:38:05.000000000 +0100
+++ seahorse-2.22.3.new/m4/gecko.m4	2008-07-10 01:03:22.000000000 +0200
@@ -51,22 +51,32 @@
 AC_MSG_CHECKING([which gecko to use])
 
 AC_ARG_WITH([gecko],
-	AS_HELP_STRING([--with-gecko@<:@=mozilla|firefox|seamonkey|xulrunner@:>@],
+	AS_HELP_STRING([--with-gecko@<:@=mozilla|firefox|seamonkey|xulrunner|libxul-unstable@:>@],
 		       [Which gecko engine to use (autodetected by default)]))
 
 # Backward compat
 AC_ARG_WITH([mozilla],[],[with_gecko=$withval],[])
 
 gecko_cv_gecko=$with_gecko
+if $PKG_CONFIG --exists $with_gecko-xpcom; then
+	gecko_cv_gecko_xpcom=$with_gecko-xpcom
+else
+	gecko_cv_gecko_xpcom=$with_gecko
+fi              
 
 # Autodetect gecko
-_geckos="firefox mozilla-firefox seamonkey mozilla xulrunner"
+_geckos="firefox mozilla-firefox seamonkey mozilla xulrunner libxul-unstable"
 if test -z "$gecko_cv_gecko"; then
 	for lizard in $_geckos; do
 		if $PKG_CONFIG --exists $lizard-xpcom; then
 			gecko_cv_gecko=$lizard
+			gecko_cv_gecko_xpcom=$lizard-xpcom
 			break;
-		fi
+		elif $PKG_CONFIG --exists $lizard; then
+		        gecko_cv_gecko=$lizard
+		        gecko_cv_gecko_xpcom=$lizard
+		        break;                  
+                fi              
 	done
 fi
 
@@ -93,11 +103,12 @@
 seamonkey) gecko_cv_gecko_flavour=mozilla ;;
 *firefox) gecko_cv_gecko_flavour=toolkit ;;
 xulrunner) gecko_cv_gecko_flavour=toolkit ;;
+libxul-unstable) gecko_cv_gecko_flavour=toolkit ;;
 esac
 
-_GECKO_INCLUDE_ROOT="`$PKG_CONFIG --variable=includedir ${gecko_cv_gecko}-xpcom`"
-_GECKO_HOME="`$PKG_CONFIG --variable=libdir ${gecko_cv_gecko}-xpcom`"
-_GECKO_PREFIX="`$PKG_CONFIG --variable=prefix ${gecko_cv_gecko}-xpcom`"
+_GECKO_INCLUDE_ROOT="`$PKG_CONFIG --variable=includedir ${gecko_cv_gecko_xpcom}`"
+_GECKO_HOME="`$PKG_CONFIG --variable=libdir ${gecko_cv_gecko_xpcom}`"
+_GECKO_PREFIX="`$PKG_CONFIG --variable=prefix ${gecko_cv_gecko_xpcom}`"
 
 fi # if gecko_cv_have_gecko
 
@@ -112,6 +123,7 @@
 $1[]_INCLUDE_ROOT=$_GECKO_INCLUDE_ROOT
 $1[]_HOME=$_GECKO_HOME
 $1[]_PREFIX=$_GECKO_PREFIX
+$1[]_XPCOM=$gecko_cv_gecko_xpcom
 
 # **************************************************************
 # This is really gcc-only
@@ -354,10 +366,10 @@
 _SAVE_CXXFLAGS="$CXXFLAGS"
 _SAVE_LDFLAGS="$LDFLAGS"
 _SAVE_LIBS="$LIBS"
-CPPFLAGS="$CPPFLAGS $_GECKO_EXTRA_CPPFLAGS -I$_GECKO_INCLUDE_ROOT $($PKG_CONFIG --cflags-only-I ${gecko_cv_gecko}-xpcom)"
-CXXFLAGS="$CXXFLAGS $_GECKO_EXTRA_CXXFLAGS $($PKG_CONFIG --cflags-only-other ${gecko_cv_gecko}-xpcom)"
+CPPFLAGS="$CPPFLAGS $_GECKO_EXTRA_CPPFLAGS -I$_GECKO_INCLUDE_ROOT $($PKG_CONFIG --cflags-only-I ${gecko_cv_gecko_xpcom})"
+CXXFLAGS="$CXXFLAGS $_GECKO_EXTRA_CXXFLAGS $($PKG_CONFIG --cflags-only-other ${gecko_cv_gecko_xpcom})"
 LDFLAGS="$LDFLAGS $_GECKO_EXTRA_LDFLAGS -Wl,--rpath=$_GECKO_HOME"
-LIBS="$LIBS $($PKG_CONFIG --libs ${gecko_cv_gecko}-xpcom)"
+LIBS="$LIBS $($PKG_CONFIG --libs ${gecko_cv_gecko_xpcom})"
 
 _GECKO_DISPATCH_INCLUDEDIRS="$2"
 
@@ -590,11 +602,11 @@
 AC_DEFUN([GECKO_XPIDL],
 [AC_REQUIRE([GECKO_INIT])dnl
 
-_GECKO_LIBDIR="`$PKG_CONFIG --variable=libdir ${gecko_cv_gecko}-xpcom`"
+_GECKO_LIBDIR="`$PKG_CONFIG --variable=libdir ${gecko_cv_gecko_xpcom}`"
 
 AC_PATH_PROG([XPIDL],[xpidl],[no],[$_GECKO_LIBDIR:$PATH])
 
-XPIDL_IDLDIR="`$PKG_CONFIG --variable=idldir ${gecko_cv_gecko}-xpcom`"
+XPIDL_IDLDIR="`$PKG_CONFIG --variable=idldir ${gecko_cv_gecko_xpcom}`"
 
 # Older geckos don't have this variable, see
 # https://bugzilla.mozilla.org/show_bug.cgi?id=240473
diff -urN seahorse-2.22.3/plugins/epiphany/mozilla/Makefile.am seahorse-2.22.3.new/plugins/epiphany/mozilla/Makefile.am
--- seahorse-2.22.3/plugins/epiphany/mozilla/Makefile.am	2007-11-24 22:38:27.000000000 +0100
+++ seahorse-2.22.3.new/plugins/epiphany/mozilla/Makefile.am	2008-07-10 01:14:31.642990537 +0200
@@ -10,6 +10,7 @@
 	docshell	\
 	dom		\
 	gtkembedmoz	\
+	string		\
 	webbrwsr	\
 	widget		\
 	xpcom
