diff -urN seahorse-1.0.1/plugins/nautilus/seahorse-pgp-preferences.desktop.in seahorse-1.0.1.new/plugins/nautilus/seahorse-pgp-preferences.desktop.in
--- seahorse-1.0.1/plugins/nautilus/seahorse-pgp-preferences.desktop.in	2007-03-17 20:36:55.000000000 +0100
+++ seahorse-1.0.1.new/plugins/nautilus/seahorse-pgp-preferences.desktop.in	2007-04-08 11:49:10.000000000 +0200
@@ -6,7 +6,7 @@
 Terminal=false
 Type=Application
 Icon=seahorse-preferences
-Categories=GNOME;GTK;Settings;
+Categories=GNOME;GTK;Settings;X-Capplet;
 StartupNotify=true
 X-GNOME-Bugzilla-Bugzilla=GNOME
 X-GNOME-Bugzilla-Product=seahorse
