--- seahorse-0.7.4/libseahorse/Makefile.am.orig	2004-10-15 00:12:08.000000000 +0200
+++ seahorse-0.7.4/libseahorse/Makefile.am	2005-02-13 21:41:14.164943886 +0100
@@ -11,12 +11,12 @@
 	-DLOCALEDIR=\"$(localedir)\" \
 	-DPIXMAPSDIR=\""$(pixmapsdir)"\"
 
-noinst_LIBRARIES = libseahorse.a
+noinst_LTLIBRARIES = libseahorse.la
 
 BUILT_SOURCES = \
 	seahorse-marshal.c seahorse-marshal.h
 
-libseahorse_a_SOURCES = \
+libseahorse_la_SOURCES = \
 	seahorse-key.c seahorse-key.h \
 	seahorse-key-pair.c seahorse-key-pair.h \
 	seahorse-context.c seahorse-context.h \
--- seahorse-0.7.4/src/Makefile.am.orig	2004-10-14 04:46:18.000000000 +0200
+++ seahorse-0.7.4/src/Makefile.am	2005-02-13 21:43:09.259640779 +0100
@@ -34,7 +34,7 @@
 	seahorse-process-multiple.c
 
 seahorse_LDADD = \
-	$(top_builddir)/libseahorse/libseahorse.a \
+	$(top_builddir)/libseahorse/libseahorse.la \
 	$(SEAHORSE_LIBS)
 
 glade_DATA = \
--- seahorse-0.7.4/plugins/nautilus/Makefile.am.orig	2005-02-13 21:36:35.480412000 +0100
+++ seahorse-0.7.4/plugins/nautilus/Makefile.am	2005-02-13 21:42:49.589986545 +0100
@@ -14,7 +14,7 @@
 	seahorse-pgp-preferences-cache.c
 	
 seahorse_pgp_preferences_LDADD = \
-	$(top_builddir)/libseahorse/libseahorse.a \
+	$(top_builddir)/libseahorse/libseahorse.la \
 	$(SEAHORSE_LIBS)
 
 # The old mime structure 
--- seahorse-0.7.4/plugins/gedit/Makefile.am.orig	2004-10-14 19:26:27.000000000 +0200
+++ seahorse-0.7.4/plugins/gedit/Makefile.am	2005-02-13 21:42:37.628417530 +0100
@@ -13,7 +13,7 @@
 libseahorse_pgp_la_SOURCES = seahorse-gedit.c
 libseahorse_pgp_la_LDFLAGS = $(PLUGIN_LIBTOOL_FLAGS)
 libseahorse_pgp_la_LIBADD = \
-	$(top_builddir)/libseahorse/libseahorse.a \
+	$(top_builddir)/libseahorse/libseahorse.la \
 	$(SEAHORSE_LIBS)
 
 plugin_in_files = seahorse-pgp.gedit-plugin.desktop.in
--- seahorse-0.7.4/agent/Makefile.am.orig	2004-10-15 00:12:08.000000000 +0200
+++ seahorse-0.7.4/agent/Makefile.am	2005-02-13 21:41:43.666267369 +0100
@@ -21,7 +21,7 @@
                          eggtrayicon.h eggtrayicon.c
 
 seahorse_agent_LDADD = \
-	$(top_builddir)/libseahorse/libseahorse.a \
+	$(top_builddir)/libseahorse/libseahorse.la \
 	$(SEAHORSE_LIBS)
                          
 glade_DATA = \
