diff -aurN seahorse-0.7.7.orig/plugins/nautilus/seahorse-pgp-preferences.desktop.in seahorse-0.7.7/plugins/nautilus/seahorse-pgp-preferences.desktop.in
--- seahorse-0.7.7.orig/plugins/nautilus/seahorse-pgp-preferences.desktop.in	2005-03-23 04:55:07.000000000 +0100
+++ seahorse-0.7.7/plugins/nautilus/seahorse-pgp-preferences.desktop.in	2005-04-16 08:34:34.000000000 +0200
@@ -5,7 +5,7 @@
 Terminal=false
 Type=Application
 Icon=gnome-mime-application-pgp-keys.png
-Categories=Application;Settings;Advanced
+Categories=GTK;GNOME;Settings;AdvancedSettings;X-Capplet;
 StartupNotify=true
 X-GNOME-Bugzilla-Bugzilla=GNOME
 X-GNOME-Bugzilla-Product=seahorse
diff -aurN seahorse-0.7.7.orig/src/seahorse.desktop.in seahorse-0.7.7/src/seahorse.desktop.in
--- seahorse-0.7.7.orig/src/seahorse.desktop.in	2005-03-23 05:17:08.000000000 +0100
+++ seahorse-0.7.7/src/seahorse.desktop.in	2005-04-16 08:36:06.000000000 +0200
@@ -5,7 +5,7 @@
 Terminal=false
 Type=Application
 Icon=seahorse.png
-Categories=Application;Cryptography;
+Categories=GTK;GNOME;Utility;
 StartupNotify=true
 X-GNOME-Bugzilla-Bugzilla=GNOME
 X-GNOME-Bugzilla-Product=seahorse
