diff -aurN seahorse-0.7.7.orig/src/seahorse-keyserver-search.glade seahorse-0.7.7/src/seahorse-keyserver-search.glade
--- seahorse-0.7.7.orig/src/seahorse-keyserver-search.glade	2005-04-05 23:14:25.000000000 +0200
+++ seahorse-0.7.7/src/seahorse-keyserver-search.glade	2005-04-20 12:45:30.000000000 +0200
@@ -312,6 +312,7 @@
 		  <property name="has_frame">True</property>
 		  <property name="invisible_char">*</property>
 		  <property name="activates_default">False</property>
+		  <signal name="changed" handler="search_changed" last_modification_time="Mon, 18 Apr 2005 21:35:11 GMT"/>
 		</widget>
 		<packing>
 		  <property name="padding">0</property>
