diff -aurN seahorse-0.8.1.orig/plugins/nautilus/seahorse-pgp-preferences.desktop.in seahorse-0.8.1/plugins/nautilus/seahorse-pgp-preferences.desktop.in
--- seahorse-0.8.1.orig/plugins/nautilus/seahorse-pgp-preferences.desktop.in	2006-01-26 11:45:01.000000000 +0100
+++ seahorse-0.8.1/plugins/nautilus/seahorse-pgp-preferences.desktop.in	2006-03-06 12:14:04.228625632 +0100
@@ -6,7 +6,7 @@
 Terminal=false
 Type=Application
 Icon=gnome-mime-application-pgp-keys.png
-Categories=GNOME;GTK;Application;Settings;Advanced;
+Categories=GTK;GNOME;Settings;AdvancedSettings;X-Capplet;
 StartupNotify=true
 X-GNOME-Bugzilla-Bugzilla=GNOME
 X-GNOME-Bugzilla-Product=seahorse
diff -aurN seahorse-0.8.1.orig/src/seahorse.desktop.in seahorse-0.8.1/src/seahorse.desktop.in
--- seahorse-0.8.1.orig/src/seahorse.desktop.in	2006-01-26 12:05:09.000000000 +0100
+++ seahorse-0.8.1/src/seahorse.desktop.in	2006-03-06 12:14:53.335160304 +0100
@@ -6,7 +6,7 @@
 Terminal=false
 Type=Application
 Icon=seahorse.png
-Categories=GNOME;GTK;Application;Cryptography;
+Categories=GNOME;GTK;Utility;
 StartupNotify=true
 X-GNOME-Bugzilla-Bugzilla=GNOME
 X-GNOME-Bugzilla-Product=seahorse
